test/webapps/bookstore-spring/src/main/java/com/springapp/mvc/extensions/WebRequestRunIdTelemetryInitializer.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Why is that?
Isn't the only dependency is in the applicationinsights.xml configuration file, i.e. there is no compile time dependency?
Even if there is you can make this **test** application dependent on the test framework.